README.md

```diff
@@ -137,20 +129,34 @@ To run the project:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## ❓ FAQ (QUESTIONS) <a name="faq"></a>
+## 🔭 Future Features <a name="future-features"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[N/A]**
+- [ ] **[N/A]**
+- [ ] **[N/A]**
```

Comment:
[OPTIONAL] You have done well in making the changes that the previous reviewer requested 👍. I would like to suggest that you add about three features that you would like to add to your project in this section of the readme. This adds a professional touch to your documentation and shows any prospective recruiter that you show your project to that you are development-oriented and are keen on improving your projects to make them better. I mark this as optional only because the previous reviewer didn't mention it directly when he talked about the future features header but I really do encourage you to add it and also add it in your future projects 👍.